src/styles/main.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables";
+@import "./utils/normalize";
+
+@import "./blocks/card.scss";
+@import "./blocks/container.scss";
```

Comment:
Change the file import to './blocks/container', as the imported file is an SCSS file, and the extension can be omitted.